packages/design-system/src/styles/components/_VerticalNav.scss

```diff
@@ -100,9 +132,39 @@
 
   .ds-c-vertical-nav__label--current {
     border-color: transparent;
+
+    /* stylelint-disable scss/media-feature-value-dollar-variable */
+    @media (-ms-high-contrast: active), (forced-colors: active) {
+      border-color: WindowText;
+    }
+    /* stylelint-enable scss/media-feature-value-dollar-variable */
   }
 
   .ds-c-vertical-nav__subnav .ds-c-vertical-nav__label {
     padding-left: $spacer-4;
   }
 }
+
+/* stylelint-disable */
```

Comment:
I also placed this gnarly HCM-only styling at the end of the Sass partial because I was feeling like its addition, in addition to the Stylelint comments, was negatively affecting the partials legibility.